test/quantization/fx/test_quantize_fx.py

```diff
@@ -4771,6 +4776,155 @@ def _test(prepare_fn, qconfig_dict):
         _test(prepare_fx, get_default_qconfig_mapping())
         _test(prepare_qat_fx, get_default_qat_qconfig_mapping())
 
+    def test_backend_config_quantization_range(self):
+        """
+        Check that quantization ranges specified through the BackendConfig are reflected in
+        the observers inserted into the model.
+        """
+        class MyModel(torch.nn.Module):
+            def __init__(self):
+                super(MyModel, self).__init__()
+                self.linear = torch.nn.Linear(30, 4).float()
+
+            def forward(self, x):
+                return self.linear(x)
+
+        dtype_config = DTypeConfig(
+            input_dtype_with_constraints=DTypeWithConstraints(
```

Comment:
do we need a new field or should we just merge with the existing `input_dtype`?

Reply:
Oh right, yes it would be cleaner. I will change this to a Union